### PR TITLE
AC Automated Test - LRAC-8849 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**
https://issues.liferay.com/browse/LRAC-8849

### CODE REVIEW

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by timpak** [comment]
reviewing PR

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataSourcesCSV.testcase [hunk 1]
```diff
@@ -31,12 +31,81 @@ definition {
...
+			ACNavigation.goToIndividuals();
+
+			ACNavigation.switchTab(tabName = "Known Individuals");
+
+			ACIndividualsDashboard.goToKnownIndividualDetails(individualName = "Manuel Test");
+
+			ACNavigation.switchTab(tabName = "Details");
 
-		// TODO LRAC-8849 AddCSVWithIndividualWhoseEmailMatchesAnotherIndividualAlreadyHasActivitiesCheckIndividualDetailsUpdated pending implementation
+			ACIndividualsDashboard.assertIndividualAttribute(individualAttribute = "wuksiwze@fo.pf");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataSourcesCSV.testcase:75]: Hi @LucasAcacio, Just to confirm, is this information visible even before adding the CSV?

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataSourcesCSV.testcase:75]: If you're right, for me your test is very good

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataSourcesCSV.testcase:75]: I think the asserts are okay.

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

Could you please just resolve the conflict and resend it? Thanks!

**Comment by LucasAcacio** [comment]
Conflicts solved

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**